gnoblog-cli/post.go

```diff
@@ -24,6 +25,18 @@ func parsePost(reader io.Reader) (*post, error) {
 		return nil, fmt.Errorf("invalid post: %w", err)
 	}
 	p.Body = string(rest)
+	p.Tags = removeWhitespace(p.Tags)
 
 	return &p, nil
 }
+
+// removeWhitespace loops over a slice of tags (strings) and truncates each tag
+// by removing whitespace
+// ie, []string{"example spaced tag"} -> []string{"examplespacedtag"}
+func removeWhitespace(tags []string) []string {
```

Comment:
Tag format should be clearly defined in CONTRIBUTIONS.md - we should have as many things ready for deployment from the get-go. This function can be a sanity check, but consider also removing it.

Reply:
Right, [tags](https://github.com/gnolang/blog/pull/32/files#diff-368b562d3cfa6abd883da53a223c32a9e2e0d021cc69b8f6fd129c74d93d3a1bL5) should be added correctly from the beginning but yeah, this can be a safety net in case people happen to do things differently.